src/tallies/tally_scoring.cpp

```diff
@@ -2362,11 +2330,7 @@ void score_collision_tally(Particle& p)
   // Determine the collision estimate of the flux
   double flux = 0.0;
   if (p.type() == ParticleType::neutron || p.type() == ParticleType::photon) {
-    if (!settings::survival_biasing) {
-      flux = p.wgt_last() / p.macro_xs().total;
-    } else {
-      flux = (p.wgt_last() + p.wgt_absorb()) / p.macro_xs().total;
```

Comment:
Was this previously incorrect? It seems the answer would be different in the previous setup.

Reply:
No, it was correct. The new line that replaces it is actually equivalent because now we are not modifying `wgt_last` during survival biasing (i.e., the absorbed weight was never subtracted off of `wgt_last`).